lib/gems/pending/metadata/util/win32/system_path_win.rb

```diff
@@ -95,7 +95,7 @@ def self.parse_bcd(fs)
       # Decode disk signature
       unless result[:os_device].nil?
         d = result[:os_device].split(',')
-        result[:disk_sig] = (d[59] + d[58] + d[57] + d[56]).to_i(16)
+        result[:disk_sig] = (d[59] + d[58] + d[57] + d[56]).to_i(16) if d.size > 60
```

Comment:
This change seems to be unrelated to the previous. It probably should be in a separate PR.

What happens when `d.size <= 60`? Is that a valid disk?